src/manager/evm/manager.rs

```diff
@@ -87,36 +87,22 @@ impl<M: Middleware + Send + Sync + 'static> SubnetManager for EthSubnetManager<M
         log::info!("creating subnet on evm with params: {params:?}");
 
         let call = self.registry_contract.new_subnet_actor(params);
-        let pending_tx = call.send().await?;
-        // We need the retry to parse the deployment event. At the time of this writing, it's a bug
-        // in current FEVM that without the retries, events are not picked up.
-        // See https://github.com/filecoin-project/community/discussions/638 for more info and updates.
-        let receipt = pending_tx.retries(TRANSACTION_RECEIPT_RETRIES).await?;
-        match receipt {
-            Some(r) => {
-                for log in r.logs {
-                    log::debug!("log: {log:?}");
-
-                    match ethers_contract::parse_log::<subnet_registry::SubnetDeployedFilter>(log) {
-                        Ok(subnet_deploy) => {
-                            let subnet_registry::SubnetDeployedFilter {
-                                subnet_addr,
-                                subnet_id,
-                            } = subnet_deploy;
-
-                            log::debug!("subnet with id {subnet_id:?} deployed at {subnet_addr:?}");
-                            return ethers_address_to_fil_address(&subnet_addr);
-                        }
-                        Err(_) => {
-                            log::debug!("no event for subnet actor published yet, continue");
-                            continue;
-                        }
-                    }
-                }
-                Err(anyhow!("no logs receipt"))
+        call.send().await?.await?;
+
+        // getting the address
+        let sender = match self.eth_client.default_sender() {
```

Comment:
I think it might be better to use `event` as the default implementation, because the existing method only tracks the latest deployed contract, so one deploys two subnets concurrently, we might not be able to get the correct result.